package/firmware/linux-firmware/qca_ath10k.mk

```diff
@@ -25,6 +25,22 @@ define Download/qca99x0-board-2g
 endef
 $(eval $(call Download,qca99x0-board-2g))
 
+define Download/qca99x0-board-5g-high
+  URL:=https://gitlab.zenterprise.org/zenjabba/starlink-wifi/-/raw/master/dl/wlan/halphy_tools/host/eepromUtil/release_ar900b/hw2/
```

Comment:
This is also available on github - https://github.com/SpaceExplorationTechnologies/starlink-wifi/tree/master/dl/wlan/halphy_tools/host/eepromUtil/release_ar900b/hw2